precise_driver/include/precise_driver/pflex_device.h

```diff
@@ -60,9 +60,14 @@ namespace precise_driver
         int getMode();
         bool setMode(const int& mode);
         std::string command(const std::string& cmd);
+        bool graspPlate(const int& width, const int& speed, const double& force);
```

Comment:
As posted here, but maybe got lost:
https://github.com/mojin-robotics/precise_ros/pull/4/files/44cd4700c078ea39d02b09a1045596e0d0f8658e#r512107219

Is this code from a third party? 
Is there any reason to pass primitive types as references that I am not aware of? By value would be better:
* No indirection of the reference. The compiler internally implements references as pointers (64bit).
* Give the compiler the hint to optimize the call since nothing else except this function will use the variable.

This is done in most of the functions inside this class. If any of these functions get called a lot, it will make a difference

Reply:
> As posted here, but maybe got lost

Got lost...

> Is this code from a third party?

Nope. I've done this...

> Is there any reason to pass primitive types as reference

No, no special reason for passing primitive types as reference. Just my laziness and a bit rusty c++ produced this ;-) You are absolutely right, thanks for the hint.

Happy you peeked into this code. Comments always welcome